frontend/packages/insights-plugin/src/components/InsightsPopup/index.tsx

```diff
@@ -55,12 +60,14 @@ export const InsightsPopup: React.FC<PrometheusHealthPopupProps> = ({ responses,
           'insights-plugin~Insights Advisor identifies and prioritizes risks to security, performance, availability, and stability of your clusters.',
         )}
       </StackItem>
-      {isError && <StackItem>{t('insights-plugin~Temporary unavailable.')}</StackItem>}
-      {isWaitingOrDisabled && (
-        <StackItem>{t('insights-plugin~Disabled or waiting for results.')}</StackItem>
-      )}
-      <StackItem>
-        {!isWaitingOrDisabled && !isError && (
+      {isError ? (
+        <StackItem>{t('insights-plugin~Temporary unavailable.')}</StackItem>
```

Comment:
```suggestion
        <StackItem>{t('insights-plugin~Temporarily unavailable.')}</StackItem>
```

Reply:
I wonder if we should apply a style like `text-muted` to differentiate between the help text and status. Have we gotten UX input?

Reply:
@spadgett, thanks, this is a very good tip. And yes, in fact, we have this field grayed out in the mockups from UX, and this class name helped to easily achieve it.